client/src/components/SearchableDropdownField/SearchableDropdownField.js

```diff
@@ -92,7 +92,11 @@ const SearchableDropdownField = ({
       setHasChanges(true);
       setJustChanged(true);
     }
-    onChange(val);
+    if (multi) {
+      onChange(val);
```

Comment:
Please update the case where the value is multi, the same as the non-multi case.
There's code to test this in https://github.com/silverstripe/silverstripe-admin/pull/1736#issuecomment-2099568462